packages/ui/components/form/inputs/Input.tsx

```diff
@@ -328,7 +328,7 @@ type FormProps<T extends object> = { form: UseFormReturn<T>; handleSubmit: Submi
   "onSubmit"
 >;
 
-const PlainForm = <T extends FieldValues>(props: FormProps<T>, ref: Ref<HTMLFormElement>) => {
+const PlainForm = (props: FormProps<{ apiError: string }>, ref: Ref<HTMLFormElement>) => {
```

Comment:
I updated this `FormProps<T>`  to `FormProps<{ apiError: string }>`  if I don't do this the typescript will show a warning here `form.clearErrors("apiError");` in line 343. So my doubt is will this cause any other issues?

Reply:
Yes it will, this isn't the right fix unfortunately - we can look at creating a "globalError" utility in the future but by not forwarding the T extends FieldValues it will disable/cause type errors elsewhere. This should be handled in the change password form. 🙏